_plugins/glossary.rb

```diff
@@ -0,0 +1,74 @@
+module EITI
+
+  module Glossary
+
+    def term(str, term = nil)
+      ##
+      # Converts a string to a glossary link with book icon.
+      #
+      # Usage:
+      # {{ "word" | term }} >>>
+      # <span class="term"
+      #       data-term="word"
+      #       title="Click to define"
+      #       tabindex="0"
+      #       style="margin-right:5px">
+      #   word
+      #   <icon class="icon-book"></icon>
+      # </span>
+      # {{ "word" | term:"alternative def" }} >>>
+      # <span class="term"
+      #       data-term="alternative def"
+      #       title="Click to define"
+      #       tabindex="0"
+      #       style="margin-right:5px">
+      #   word
+      #   <icon class="icon-book"></icon>
+      # </span>
+      unless term
+        return "<span class='term' data-term='#{str}' \
+          title='Click to define' tabindex='0'>#{str}<icon \
+          class='icon-book'></icon></span>"
+      else
+        return "<span class='term' data-term='#{term}' \
+          title='Click to define' tabindex='0'>#{str}<icon \
+          class='icon-book'></icon></span>"
+      end
+    end
+
+    def term_end(str, term = nil)
+      ##
+      # Same as term, but eliminates margin to the right of the book icon
+      #
+      # Usage:
+      # {{ "word" | term }} >>>
+      # <span class="term"
+      #       data-term="word"
+      #       title="Click to define"
+      #       tabindex="0">
+      #   word
+      #   <icon class="icon-book"></icon>
+      # </span>
+      # {{ "word" | term:"alternative def" }} >>>
+      # <span class="term"
+      #       data-term="alternative def"
+      #       title="Click to define"
+      #       tabindex="0" >
+      #   word
+      #   <icon class="icon-book"></icon>
+      # </span>
+      unless term
```

Comment:
Personally, I prefer a negative to a double positive, but if people have strong objections, speak now!